02a-dbc.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: page
+title: Testing
+subtitle: Design by Contract
+minutes: 15
+---
+> ## Learning Objectives {.objectives}
+>
+> *   Design by Contract is a way of using Assertions for interface specification.
+> *   Design by Contract defines pre-, post- and invariant conditions a function and a caller agree to obey.
```

Comment:
This objective might be more parsable to a beginner if it's more granular. For example, understanding the meaning of "pre-, post- and invariant conditions" in this context could be its own objective.